modules/Sources/Features/Root/RootInitialization.swift

```diff
@@ -42,11 +42,14 @@ extension RootReducer {
                 state.appStartState = .didFinishLaunching
                 // TODO: [#704], trigger the review request logic when approved by the team,
                 // https://github.com/Electric-Coin-Company/zashi-ios/issues/704
-                return .run { send in
-                    try await mainQueue.sleep(for: .seconds(0.5))
-                    await send(.initialization(.initialSetups))
-                }
-                .cancellable(id: DidFinishLaunchingId.timer, cancelInFlight: true)
+                return .concatenate(
+                    Effect.send(.initialization(.configureCrashReporter)),
+                    .run { send in
+                        try await mainQueue.sleep(for: .seconds(0.5))
```

Comment:
Can we do without this additional sleep here? What's its purpose? Thanks.

Reply:
We can't. When Zashi runs in the background, we need to cancel this action before it's executed.